### ISSUE TITLE
Unpack 7zip archives (renamed as tar.xz)

### ISSUE THREAD

**Issue by kenderipa** [marcosnils/bin#209]
Hello, could you add the recognition of 7zip archive structure, even renamed to tar.xz? Apparently at least Forkgram (https://github.com/forkgram/tdesktop) apparently provides its release in this format. Thanks!

**Comment by needleshaped** [comment]
I would argue, that this mut be registered as issue on that project, and solved upstream, instead of making workaround downstream.

**Comment by kenderipa** [comment]
> I would argue, that this mut be registered as issue on that project, and solved upstream, instead of making workaround downstream.

There are a lot of archivers with different algorythms, running between all the software developers on github and trying to convince them to use tar seems a bit counterproductive. I think that bin should be able to use at least several most common ones. After all, bin is here to make our lives easier, not the other way around.

**Comment by marcosnils** [comment]
> I think that bin should be able to use at least several most common ones. After all, bin is here to make our lives easier, not the other way around.

 yes, agree. Should be relatively issue to support. Sorry, I haven't had too much time in my hands for OSS personal projects lately 🙏

**Comment by kenderipa** [comment]
> > I think that bin should be able to use at least several most common ones. After all, bin is here to make our lives easier, not the other way around.
> 
> yes, agree. Should be relatively issue to support. Sorry, I haven't had too much time in my hands for OSS personal projects lately 🙏

Of course, take your time.
